### ISSUE TITLE
[Request] Health/AA decorator for target nodes

### ISSUE THREAD

**Issue by tthvo** [cryostatio/cryostat-web#931]
## Describe the feature

Health check decorators for target nodes. This could be the result of automatic analysis report.

## Any other information?

We can add it in 2 of the remaining quadrant in target nodes.

**Comment by tthvo** [comment]
Maybe, there could be an action to create Automated Analysis on the target/group and some settings for refresh intervals or just a button to refresh (+ some expire warning).

For point 3, I am not so sure of anyway to combine results into a single one but is it possible? Or we have to define ourselves?

**Comment by roberttoyonaga** [comment]
What if the badge represents an active recording (if one exists), otherwise the most recent recording?  Otherwise, no badge at all. 

Would it be reasonable to say any "red" automated analysis results  = red badge, otherwise any "yellow" results = yellow badge, otherwise green badge?

**Comment by maxcao13** [comment]
> Would it be reasonable to say any "red" automated analysis results = red badge, otherwise any "yellow" results = yellow badge, otherwise green badge?
> 

Makes sense, this is what already happens for the automated analysis card.

Generating reports on hundreds of targets at the same time definitely won't scale. What about setting some config in the topology view to say, "automatically start a report on this target perdioically" so it's an opt in feature for targets that you specifically want to have at-a-glance monitoring for?

**Comment by andrewazores** [comment]
That sounds like it could be an enhancement to Automated Rules which we can also expose some UI for within Topology.
